src/app/mission/[id]/detail/page.tsx

```diff
@@ -1,19 +1,34 @@
 'use client';
 
+import { useRouter } from 'next/navigation';
 import MissionHistoryTab from '@/app/mission/[id]/detail/MissionHistoryTab';
+import Dialog from '@/components/Dialog/Dialog';
 import Header from '@/components/Header/Header';
 import Tab from '@/components/Tab/Tab';
+import useModal from '@/hooks/useModal';
 import { css } from '@styled-system/css';
 
 export default function MissionDetailPage() {
+  const { isOpen, openModal, closeModal } = useModal();
+  const router = useRouter();
   const tabs = [
     {
       tabName: '미션 내역',
       active: true,
     },
   ];
-  const handleMenuClick = () => {
-    // TODO : 수정하기로 라우팅
+
+  const handleMenuClick = (id: string) => {
+    id === 'mission-modify' ? router.replace('detail/modify') : openModal();
```

Comment:
삼항 연산자 보다 if 문으로 작성하는게 가독성이 좋다고 생각하는데 어떻게 생각하시나요?

```suggestion
    if(d === 'mission-modify') {
       router.replace('detail/modify')
       return 
    } 
    openModal();
```

Reply:
if-else 로 해뒀다가 삼항연산자로 바꿨는데,  if 한개로 해놓으면 가독성이 더 좋을 것 같아요! 수정해보겠습니다